data/analytics/events.yml

```diff
@@ -832,9 +870,8 @@
             value: The section that this link is in. The section in this case refers to the step number.
           - name: index_link
             value: The position of the link within this section.
-          - name: index_section_count
-            value: The total amount of links within this section.
         - name: index_total
+          value: The total amount of links within this section.
```

Comment:
Same as earlier - good addition, but if we can let's put it in the attribute data so it pulls through.

Reply:
@andysellick When I remove the `value`, instead it pulls in the `type`. So it just becomes `string (number)`. Is this OK?

Reply:
Yes, I think that's okay. Although I'm now questioning which is better: `string (number)` or `number (string)`. Any thoughts?

Reply:
@andysellick I think both make sense, so I don't mind!